tests/checks/cell/test_deviated_cell.py

```diff
@@ -0,0 +1,27 @@
+from frictionless import validate, checks
+
+
+# Issues
+
+
+def test_validate_deviated_cell_1066():
+    report = validate(
+        "data/issue-1066.csv",
+        checks=[
+            checks.deviated_cell(
+                ignore_fields=[
```

Comment:
What will be the result if we don't ignore these fields?

If it fails without ignoring them we might need to tweek the logic

Reply:
If all fields are added it fails. There are many other columns which have deviated value. Sure will check the data and see if they are valid alerts.

Reply:
I think the reason for it that `deviated_cell` check is just different to `deviated_value` in some aspects. I think for `deviated_cell` we need much bigger default interval to catch really obvious problems as we have in the original CSV.

For example, consider average cell size in column: 50 characters. Even 500 might ok but 5000 is what we need to report. By default it should be really forgiving otherwise we will mark invalid a lot of valid files

Reply:
yes little bit deviation also triggers the alert. Like you said, may be we can use either 5000 or deviated value which ever is greater.